.github/workflows/canary-integration-test.yml

```diff
@@ -1,5 +1,8 @@
 name: Canary integration tests
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+    - '**.go'
```

Comment:
Do we want to run the integration test if there is changes in `.py` or `.sh` file?

Reply:
Most likely yes.